### PR TITLE
fix for issue #557

### PR DESCRIPTION
**Pull request by NicholasArmour**

### CODE REVIEW

**Comment by BrentHoltsclaw** [comment]
The logic within the hal was fine, recommend printing out the returned values within the util_cmd instead.

**Comment by npmitche** [comment]
> The logic within the hal was fine, recommend printing out the returned values within the util_cmd instead.

I think I agree with Brent:
```
if( 'detect' == op ):
    self.logger.log( "[CHIPSEC] Searching for DIMMs with SPD.." )
    _dimms = _spd.detect()
    self.logger.log( "[CHIPSEC] Detected the following SPD devices:")
    for _dimm in _dimms: self.logger.log( "[CHIPSEC]     %s: 0x%02X".format(_spd.SPD_DIMMS[_dimm], _dimm) )
```

File: chipsec/hal/spd.py [hunk 1]
```diff
@@ -364,7 +364,7 @@ def detect( self ):
         _dimms = []
         for d in SPD_DIMMS:
             if self.isSPDPresent( d ): _dimms.append( d )
-        if logger().HAL: 
+        if logger().HAL:
             logger().log( "Detected the following SPD devices:" )
             for _dimm in _dimms: logger().log( "%s: 0x%02X" % (SPD_DIMMS[_dimm],_dimm) )
```
Comment (npmitche) [inline chipsec/hal/spd.py]: If you're modifying this file, might as well also update this from % to format?

Comment (ErikBjorge) [reply chipsec/hal/spd.py]: Just a white space change so no need to do anything else.

**Pull request merged**